app/components/primer/label_component.rb

```diff
@@ -5,6 +5,9 @@ module Primer
   class LabelComponent < Primer::Component
     status :beta
 
+    DEFAULT_TAG = :span
+    TAG_OPTIONS = [DEFAULT_TAG, :summary, :a, :div].freeze
```

Comment:
huh, why `summary` and `a`. Didn't we try to move off clickable labels?

Reply:
We moved off of clickable `State`, and replaced the one example in dotcom with a `Label` which seemed to be acceptable

https://github.com/primer/view_components/pull/432
https://github.com/github/github/pull/176322